ts/components/core/spacer/Spacer.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { View } from "react-native";
+import type { IOSpacer } from "../variables/IOSpacing";
+
+export type SpacerOrientation = "vertical" | "horizontal";
+
+type SpacerProps = {
+  orientation?: SpacerOrientation;
+  size?: IOSpacer;
+};
+
+/**
+Native `Spacer` component that replaces the legacy one, managed through NativeBase.
+
+@param  {SpacerOrientation} orientation 
+@param {IOSpacerType} size
+ */
+export const Spacer = ({
+  orientation = "vertical",
```

Comment:
Taking inspiration from other UI toolkits, in addition of an `orientation` prop we could also export two different components for each orientation, like `HSpacer` and `VSpacer`, which add the `orientation` prop implicitly. I think it would become more straight forward to use:
```js
<HSpacer size={10} />
``` 
instead of 

```js
<Spacer orientation="horizontal" size={10} />
```
What you think?